index.html

```diff
@@ -0,0 +1,142 @@
+          <li class="items item1"><a href="#programs" class="nav-link program-link">Program</a></li>
+          <li class="items item1"><a href="#guests" class="nav-link join-link">Speakers</a></li>
+          <li class="items"><a href="#sponsor" class="nav-link sponsor-link">Sponsor</a></li>
+          <li class="items"><a target="_blank" rel="noopener" href="https://www.uonbi.ac.ke/" class="nav-link school-link" id="school">UoNSchool</a></li>
+      </ul>
+
+    </nav>
+
+
+  </header>
+
+  <main>
+    <section class="first">
+      <div class="intro">
+        <h3>"Hello! It is Here Again"</h3>
+        <p class="title">NAIROBI INNOVATION WEEK EDITION 3 2022</p>
+        <p class="short-info">An Interactive session about innovation in Kenya and Africa brought to you by the University of Naorobi. This is to promote advanced changes in our continent.</p>
+        <p class="date">2022.04.26(WED) - 28(FRI)</p>
+        <p class="location">@ University of Nairobi, Great Court Main Campus.</p>
+      </div>
+      <div class="overle"></div>
+    </section>
+
+
+    <section class="second" id="programs">
+      <h2>Main Program</h2>
+      <hr>
+      <div class="programs">
+        <article class="lecture">
+          <img class="lecture-img" src="./imgs/lecture1.png" alt="lecture icon">
+          <h2>Lecture</h2>
+          <p>Listen to the Speakers from various organizations about the messages of inovation</p>
+        </article>
+
+        <article class="exhibition">
+          <img class="exhibition-img" src="./imgs/exhibition1.png" alt="exhibition icon">
+          <h2>Exhibition</h2>
+          <p>Appreciate various ideas from various entrepreneurs and organizations</p>
+        </article>
+
+        <article class="forum">
+          <img class="forum-img" src="./imgs/forum2.jfif" alt="forum icon">
+          <h2>Forum</h2>
+          <p>Have the oppurtunity to share your thoughts and engage us in all our platforms</p>
+        </article>
+
+        <article class="workshop">
+          <img class="workshop-img" src="./imgs/workshop1.jfif" alt="workshop icon">
+          <h2>Workshop</h2>
+          <p>Try creating your own work using open source license rather than just watching</p>
+        </article>
+
+        <article class="ignite">
+          <img class="ignite-img" src="./imgs/ignite.png" alt="ignite icon">
+          <h2>ignite</h2>
+          <p>Get oppurtunities to network with different people from all over the world.</p>
+        </article>
+      </div>
+      <button type="button" class="see-more">Join N.I.W Conference 2022</button>
+      <a class="see-less" href="#">SEE THE WHOLE PROGRAM</a>
+    </section>
+
+    <section class="third" id="guests">
+      <h2 class="featured">Featured Speakers</h2>
+      <hr>
+      <div id="wageni"></div>
+      <div id="wageni1"></div>
+    </section>
+
+
+    <section class="four" id="sponsor">
+      <h2>partner</h2>
+      <hr class="sponsor-hr">
+      <div class="sponsors">
```

Comment:
- [ ] This is optional but you can use sematic tags instead of ```<div>``` when you can, this will improve the code readability and SEO